packages/client/src/app/personal_page/components/groups/CreateGroupModal.vue

```diff
@@ -0,0 +1,105 @@
+<script setup>
+import { CustomFormKit } from '../../../shared/components/index.js';
+import { ref } from 'vue';
+import { usePersonalPageStore } from '../../stores/personal_page.store.js';
+
+/** @type {import('vue').Ref<string>} */
+let name = ref('');
+
+/** @type {import('vue').Ref<Object>} */
+let group = ref({});
+
+/** @type {import('vue').Ref<Object>} */
+const modal = ref(null);
+/** @return {Promise<void>} */
+async function createGroupRequest() {
+  const { createGroup } = usePersonalPageStore();
+  group.value = await createGroup(name.value);
+}
+
+const handleModalOpen = () => {
```

Comment:
Can't we use regular functions for these constant anonymous functions? Makes the code base a bit more consistent